app/models/bgeigie_import.rb

```diff
@@ -38,6 +38,15 @@ def self.filter(query)
            OR lower(credits) LIKE :query", query: "%#{query.downcase}%")
   end
 
+  def self.by_user_name(name)
+    users = User.by_name(name)
+    where(user_id: users.map(&:id))
+  end
+
+  def self.rejected_by(name)
+    where(rejected_by: name)
```

Comment:
Don't we need to do case-insensitive search here?

Reply:
@matschaffer Since, we don't have index on `rejected_by` column, filtering with this column would take long time to query. Should we create index on this column or have separate table?